src/model/HeatPumpPlantLoopEIRCooling.hpp

```diff
@@ -0,0 +1,183 @@
+
+    boost::optional<double> referenceLoadSideFlowRate() const;
+
+    bool isReferenceLoadSideFlowRateAutosized() const;
+
+    boost::optional<double> referenceSourceSideFlowRate() const;
+
+    bool isReferenceSourceSideFlowRateAutosized() const;
+
+    boost::optional<double> referenceCapacity() const;
+
+    bool isReferenceCapacityAutosized() const;
+
+    double referenceCoefficientofPerformance() const;
+
+    double sizingFactor() const;
+
+    Curve capacityModifierFunctionofTemperatureCurve() const;
+
+    Curve electricInputtoOutputRatioModifierFunctionofTemperatureCurve() const;
+
+    Curve electricInputtoOutputRatioModifierFunctionofPartLoadRatioCurve() const;
+
+    //@}
+    /** @name Setters */
+    //@{
+
+    bool setCondenserType(const std::string& condenserType);
+
+    bool setCompanionHeatingHeatPump(const HeatPumpPlantLoopEIRHeating& companionHP);
+
+    bool setReferenceLoadSideFlowRate(double referenceLoadSideFlowRate);
+
+    void autosizeReferenceLoadSideFlowRate();
+
+    bool setReferenceSourceSideFlowRate(double referenceSourceSideFlowRate);
+
+    void autosizeReferenceSourceSideFlowRate();
+
+    bool setReferenceCapacity(double referenceCapacity);
+
+    void autosizeReferenceCapacity();
+
+    bool setReferenceCoefficientofPerformance(double referenceCoefficientofPerformance);
+
+    bool setSizingFactor(double sizingFactor);
+
+    bool setCapacityModifierFunctionofTemperatureCurve(const Curve& capacityModifierFunctionofTemperatureCurve);
+
+    bool setElectricInputtoOutputRatioModifierFunctionofTemperatureCurve(const Curve& electricInputtoOutputRatioModifierFunctionofTemperatureCurve);
+
+    bool
+      setElectricInputtoOutputRatioModifierFunctionofPartLoadRatioCurve(const Curve& electricInputtoOutputRatioModifierFunctionofPartLoadRatioCurve);
+
+    //@}
+    /** @name Other */
+    //@{
+
+    boost::optional<double> autosizedReferenceLoadSideFlowRate() const;
+
+    boost::optional<double> autosizedReferenceSourceSideFlowRate() const;
+
+    boost::optional<double> autosizedReferenceCapacity() const;
+
+    /** Convenience Function to return the Load Side Water Loop (HeatPump on supply side) **/
+    boost::optional<PlantLoop> loadSideWaterLoop() const;
+
+    /** Convenience Function to return the Source Side (Condenser) Water Loop (HeatPump on demand side) **/
+    boost::optional<PlantLoop> sourceSideWaterLoop() const;
+
+    // Convenience function to return the inletNode on the Source Side
+    boost::optional<Node> sourceSideWaterInletNode() const;
+    // Convenience function to return the outletNode on the Source Side
+    boost::optional<Node> sourceSideWaterOutletNode() const;
+
+    // Convenience function to return the inletNode on the Load Side
+    boost::optional<Node> loadSideWaterInletNode() const;
+    // Convenience function to return the outletNode on the Load Side
+    boost::optional<Node> loadSideWaterOutletNode() const;
```

Comment:
New convenience functions, as I find it terribly awkward to have to remember that load Side = the loop where this is on the supply side, and source side = the loop where it's on the demand side (a condenser loop)

I will not duplicate my comments to the Heating one, but I've done exactly the same changes